subsys/bluetooth/host/gatt.c

```diff
@@ -2030,23 +2038,18 @@ static uint8_t notify_cb(const struct bt_gatt_attr *attr, uint16_t handle,
 	return BT_GATT_ITER_CONTINUE;
 }
 
-struct find_data {
-	const struct bt_gatt_attr *attr;
-	uint16_t handle;
-};
-
 static uint8_t match_uuid(const struct bt_gatt_attr *attr, uint16_t handle,
 			  void *user_data)
 {
-	struct find_data *found = user_data;
+	struct notify_data *data = user_data;
```

Comment:
Not sure I understand the reason for changing this here.

I think I would have preferred to use find data inside notify_data, don't need rest of the notify_data in this one.

```
struct notify_data {
    struct find_data found_chrc;
    /* ... */
};
```